sources/host-ctr/go.mod

```diff
@@ -3,16 +3,31 @@ module host-ctr
 go 1.12
```

Comment:
We have Go 1.16 in the SDK so I think it'd be reasonable to bump this up.  `go mod tidy` has some behavioral changes depending on the Go version declared here (though I think the larger ones are for 1.17 rather than 1.16).